androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/current/currentpatients/PatientItem.kt

```diff
@@ -0,0 +1,142 @@
+package com.EENX15_22_17.digital_journal.android.ui.current.currentpatients
+
```

Comment:
Dont like the rename of the file. I like it better like it was before "CurrentPatientItem"